client/src/game/ui/menu/MenuBar.vue

```diff
@@ -93,6 +93,10 @@ const openClientSettings = (): void => uiSystem.showClientSettings(!uiState.raw.
         <div style="width: 12.5rem; overflow-y: auto; overflow-x: hidden">
             <!-- CHARACTERS -->
             <Characters />
+            <!-- NOTES -->
```

Comment:
I would prefer to keep the location it has in the DM menu (i.e. below assets), as for me as a dm the location of the assets section, both because it's one of the most used ones and people might have some muscle memory where to click, but also because the list can expand down vertically more when it's higher.

I do agree that notes should be visible to players though, so could you end the `dm if clause` before notes and re-open it afterwards instead?

Reply:
Totally makes sense, will do